src/cljc/commiteth/model/bounty.cljc

```diff
@@ -0,0 +1,35 @@
+(ns commiteth.model.bounty
+  (:require [commiteth.util :as util]))
+
+;; Most of the functions in here are currently intended for use inside the CLJS codebase
+;; record maps look vastly different on frontend and backend due to simple things like kebab/camel
+;; casing as well as more complex stuff like Postgres views shuffling data around
+
+;; In the future we may want to establish Clojure records to assign names to the various
+;; incarnations of maps we currently have adding the following functions to those records
+;; via a protocol. Clojure records could also be serialized via transit making it easier
+;; to communicate what datatypes are returned where.
+
+
+(defn merged? [claim]
+  (= 1 (:pr_state claim)))
+
+(defn paid? [claim]
+  (not-empty (:payout_hash claim)))
+
+(defn bot-confirm-unmined? [bounty]
+  (empty? (:confirm_hash bounty)))
+
+(defn confirming? [bounty]
+  (:confirming? bounty))
+
+(defn issue-url
```

Comment:
I believe `pr-url` lived near this in one location so that would also be a ripe candidate for this namespace or at least warrant a TODO.

Reply:
`pr-url` technically belongs to a claim so this might be a reason I didn't add it to this namespace. Should just establish that namespace I guess.